Client/Frontend/Settings/SettingsViewController.swift

```diff
@@ -190,15 +190,15 @@ class SettingsViewController: TableViewController {
             ]
         }
         
-        vpnRow = vpnSettingsRow()
-        
         section.rows.append(
             Row(text: Strings.BraveToday.braveToday, selection: {
                 let todaySettings = BraveTodaySettingsViewController(dataSource: self.feedDataSource)
                 self.navigationController?.pushViewController(todaySettings, animated: true)
             }, image: #imageLiteral(resourceName: "settings-brave-today").template, accessory: .disclosureIndicator)
         )
         
+        section.rows.append(vpnSettingsRow())
+        
```

Comment:
You aren't setting `vpnRow` anymore, doesn't that need to happen?

Reply:
good catch, it's needed to update the label with `enabled/disabled/expired` states